data/common/include/MercuryDefinitions.h

```diff
@@ -30,10 +30,18 @@ namespace Mercury {
 
     typedef struct
     {
-      uint32_t frame_counter;
-      uint32_t packet_number_flags;
+    	uint32_t frame_counter;

```

Comment:
Indentation is inconsistent in header file - seems some are tabs, some are spaces. Should be all spaces

Reply:
Apologies, I did implement the requested changes in commit 08bea7d but I failed to merge and close the PR.